web/src/constants/timers.js

```diff
@@ -0,0 +1 @@
+export const slowLoadingThreshold = 5000;
```

Comment:
Customer has a /dashboard that takes over a second to load. Bumping this up to 5 seconds for now. I think long term we'll speed up the endpoint and remove the full page loader for these polling fetches.

Reply:
What would be a way to speed up the endpoint?

Reply:
Would take some backend work. I'm not sure what `/dashboard` is doing but we'd probably have to profile the tasks it's doing and try to tackle the biggest contributor to response time.